routes/project/location/load_balancer.rb

```diff
@@ -118,6 +123,15 @@ class Clover
         lb.incr_refresh_cert
       end
 
+      r.post "enable-cert" do
+        authorize("LoadBalancer:edit", lb)
+        cert_enabled = typecast_params.bool!("cert_enabled")
+        cert_enabled ? lb.enable_cert_server : lb.disable_cert_server
+        audit_log(lb, "update")
+        flash["notice"] = "SSL certificates #{cert_enabled ? "enabled" : "disabled"}"
+        r.redirect lb, "/settings"
```

Comment:
This looks like a web-only route. We should probably support this in the API/SDK/CLI.

Reply:
I initially thought api/sdk/cli would just use patch route but sure. I added them.